applications/luci-app-sshtunnel/htdocs/luci-static/resources/view/sshtunnel/ssh_servers.js

```diff
@@ -135,13 +131,35 @@ return view.extend({
 	},
 });
 
-function _splitSshKeys(sshFiles) {
-	var sshKeys = {};
-	for (var i = 0; i < sshFiles.length; i++) {
-		var sshPubKeyName = sshFiles[i];
-		var sshKeyName = sshPubKeyName.substring(0, sshPubKeyName.length - 4);
-		sshKeys[sshKeyName] = '';
+function _findAllPossibleIdKeys(entries) {
+	var sshKeyNames = [];
+	for (var item of entries) {
+		if (item.type !== 'file') {
+			continue
+		}
+		// a key file should have a corresponding .pub file
+		if (item.name.endsWith('.pub')) {
+			var sshPubKeyName = item.name;
+			var sshKeyName = sshPubKeyName.substring(0, sshPubKeyName.length - 4);
+			if (!sshKeyNames.includes(sshKeyName)) {
+				sshKeyNames.push(sshKeyName)
+			}
+		} else {
+			// or at least it should start with id_ e.g. id_dropbear
+			if (item.name.startsWith('id_')) {
+				var sshKeyName = item.name;
+				if (!sshKeyNames.includes(sshKeyName)) {
+					sshKeyNames.push(sshKeyName)
+				}
+			}
+		}
 	}
-	allSshKeys = sshKeys;
-	return sshKeys;
+	return sshKeyNames;
 }
+
+function _manSshConfig(opt) {
+	return '<br />' + _('See %s.')
+			.format('<a target="_blank" href="https://manpages.ubuntu.com/manpages/noble/en/man5/ssh_config.5.html' +
```

Comment:
In a comment I added a reason:
"Use Ubuntu man pages as a more advanced: many versions and languages."

For example here is German translation:
https://manpages.ubuntu.com/manpages/noble/de/man5/ssh_config.5.html

Unfortunately there is a bug that not allow to use a `current` in a link https://bugs.launchpad.net/ubuntu-manpage-repository/+bug/680229

But I hope one day it will be fixed. Also I hope they'll add a language switcher and negotiatior.

Also the Ubuntu/Debian is most used Linux distro or at least familiar.